src/features/session_trace/aggregate/index.js

```diff
@@ -272,6 +274,8 @@ export class Aggregate extends AggregateBase {
   // Tracks the events and their listener's duration on objects wrapped by wrap-events.
   storeEvent (currentEvent, target, start, end) {
     if (this.shouldIgnoreEvent(currentEvent, target)) return
+    if (this.prevStoredEvents.has(currentEvent)) return // prevent multiple listeners of an event from creating duplicate trace nodes per occurrence
+    this.prevStoredEvents.add(currentEvent)
```

Comment:
reminder to one of us to port this over to the ST migration PR since this could be missed in the merge conflict